packages/@adobe/spectrum-css-temp/components/buttongroup/index.css

```diff
@@ -15,23 +14,32 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-ButtonGroup {
-  display: flex;
+  display: inline-flex;
+  /* Added this so it would wrap */
+  flex-wrap: wrap;
+  align-items: flex-end;
```

Comment:
should all button groups flex end? or should this be something from slots?
maybe flex end is a good default and if needed, people can do flex-start through a new class